jest/testing-library__jest-dom.d.ts

```diff
@@ -0,0 +1,529 @@
+// Delete this file when we upgrade to Emotion 11 and upgrade everything jest-related
+
+// Type definitions for @testing-library/jest-dom 5.9
+// Project: https://github.com/testing-library/jest-dom
+// Definitions by: Ernesto García <https://github.com/gnapse>
+//                 John Gozde <https://github.com/jgoz>
+//                 Seth Macpherson <https://github.com/smacpherson64>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.8
+
+/// <reference types="jest" />
+
+declare namespace jest {
+  interface Matchers<R> {
```

Comment:
This is the only difference from the main package. It could be removed when we update to Emotion 11 and all jest types use the new 2 generic `Matcher`